lib/led/rgb.js

```diff
@@ -429,19 +479,22 @@ class RGB {
    */
 
   [Animation.render](frames) {
-    return this.color(frames[0]);
+    const state = priv.get(this);
+    state.value = frames[0];
+    return this.update();
   }
 
 
 }
+
+RGB.colors = ["red", "green", "blue"];
```

Comment:
The assignment of RGB.colors was moved before Animation.keys is assigned to it. [This small PR](https://github.com/rwaldron/johnny-five/pull/1799) resolves this issue and adds a unit test for this fix.

Reply:
#1799  has been merged.

Reply:
Thanks!